app/src/main/res/layout/activity_main.xml

```diff
@@ -1,24 +1,60 @@
-<?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:padding="16dp">
+    android:layout_height="match_parent">
 
-    <TextView
-        android:id="@+id/textView"
-        android:layout_width="wrap_content"
+    <!-- Header Fragment -->
+    <androidx.fragment.app.FragmentContainerView
```

Comment:
This is the wrong use of a fragment (I'll explain it in our meeting). this would be a custom view (we'll learn about it in the future).
so for now, move the header fragment views to here.